### PR TITLE
Task 195-196: Sentiment140

### PR DESCRIPTION
**Pull request by Mihir3009**
Please review.

### CODE REVIEW

**Review by slowy07 — APPROVED** [review]

**Comment by slowy07** [comment]
thanks sir @Mihir3009 , can you solve the conflict on pull request ??

**Comment by Mihir3009** [comment]
Resolved.

File: tasks/task170_hotpotqa_answer_generation.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "Contributors": "Mihir Parmar",
     "Source": "HotpotQA (dataset link- https://hotpotqa.github.io/)",
+    "Categories": "Question Answering",
```
Comment (danyaljj) [inline tasks/task170_hotpotqa_answer_generation.json]: ??

**Comment by danyaljj** [comment]
Thanks! The task definitions look good to me. 
Other than one issue flagged earlier, looks good to me. 
Before we can merge, please pull from the main branch so that it contains all the latest tests.

File: tasks/task170_hotpotqa_answer_generation.json (hunk 1 above)
Comment (Mihir3009) [reply tasks/task170_hotpotqa_answer_generation.json]: Resolved this issue.

**Review by swarooprm — APPROVED** [review]

**Pull request merged**